### PR TITLE
fix: integration test

### PR DESCRIPTION
**Pull request by petertonysmith94**
# Summary

- Fixed the integration tests due to #2755 getting merged ([error](https://github.com/FuelLabs/fuels-ts/actions/runs/10010519715/job/27716224529#step:6:39)).
- Added type checking for `create fuels` to avoid this occurring again.
- Updated the documentation for `create fuels` commands.

# Checklist

- [ ] I **_added_** — `tests` to prove my changes
- [ ] I **_updated_** — all the necessary `docs`
- [X] I **_reviewed_** — the entire PR myself, using the GitHub UI
- [ ] I **_described_** — all breaking changes and the Migration Guide

### CODE REVIEW

File: .github/workflows/test.yaml [hunk 1]
```diff
@@ -75,7 +75,7 @@ jobs:
           wait-interval: 30
 
       - name: Run tests (with published version)
-        if: github.ref_name == 'master' || github.head_ref == 'changeset-release/master'
+        # if: github.ref_name == 'master' || github.head_ref == 'changeset-release/master'
```
Comment (petertonysmith94) [inline .github/workflows/test.yaml:78]:
```suggestion
        if: github.ref_name == 'master' || github.head_ref == 'changeset-release/master'
```

**Comment by nedsalk** [comment]
Could you please reference the error in the PR's description so that we know what this PR is fixing? I'm out of the loop on this one.

**Review by nedsalk — APPROVED** [review]

**Review by danielbate — APPROVED** [review]

File: packages/create-fuels/package.json [hunk 2]
```diff
@@ -15,7 +15,7 @@
   },
   "license": "Apache-2.0",
   "scripts": {
-    "build": "tsup",
+    "build": "tsup && tsc --noEmit",
```
Comment (arboleya) [inline packages/create-fuels/package.json:18]: Why the `tsc` addition?

**Review by arboleya — APPROVED** [review]

File: packages/create-fuels/package.json (hunk 2 above)
Comment (petertonysmith94) [reply packages/create-fuels/package.json:18]: I added this for type checking.

Comment (arboleya) [reply packages/create-fuels/package.json:18]: 🚢

**Review by Torres-ssf — APPROVED** [review]

**Pull request merged**